README.rst

```diff
@@ -37,6 +37,19 @@ or alternatively, install js-slang and run
   $ npm -g install js-slang   # Install js-slang
   $ js-slang [chapter] # default: 1
 
+Documentation
+-------------
+
+Source is documented here: https://sicp.comp.nus.edu.sg
+
+To build the documentation, run
+
+..code-block::
+
+  $ git clone https://github.com/source-academy/js-slang.git
+  $ cd js-slang
+  $ make           # to make the PDF documents using LaTeX
+  $ yarn run jsdoc # to make the web pages in js-slang/doc/source
```

Comment:
I ran `yarn run jsdoc` and it fails with the following error:
```
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/array.js
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/pairmutator.js
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/array.js
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/pairmutator.js
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/mce.js
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/array.js
There are no input files to process.
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/pairmutator.js
There are no input files to process.
ERROR: Unable to find the source file or directory /home/chrisjwelly/projects/cloned-source-academy/js-slang/doc/mce.js
```
I tried both in the `master` and `lib-doc` branch with the same error.

I also cannot run `make` with the error: `make: *** No targets specified and no makefile found.  Stop.`